CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 - Fix an issue where required product list options would display an invalid "none" choice [#929](https://github.com/bigcommerce/stencil/pull/929)
 - Remove unused variable causing js error with search in the nav [#938](https://github.com/bigcommerce/stencil/pull/938)
 - Add settings to theme editor schema to customize Optimized Checkout discount banners [#924] (https://github.com/bigcommerce/stencil/pull/924)
+- Add optimized checkout feature [#880] (https://github.com/bigcommerce/stencil/pull/880)
```

Comment:
This sounds like you are adding the entire functionality in this PR, but is actually just a flag and this could confuse some people. Please elaborate or just add `flag` at the end